src/lightning/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""Root package info."""
+import logging
+from typing import Any
+
+_DETAIL = 15  # between logging.INFO and logging.DEBUG, used for logging in production use cases
+
+
+def _detail(self: Any, message: str, *args: Any, **kwargs: Any) -> None:
+    if self.isEnabledFor(_DETAIL):
+        # logger takes its '*args' as 'args'
+        self._log(_DETAIL, message, args, **kwargs)
+
+
+logging.addLevelName(_DETAIL, "DETAIL")
+logging.detail = _detail
+logging.Logger.detail = _detail
+
+_root_logger = logging.getLogger()
```

Comment:
I don't think that the wrapper should be defining logging logic. Where would we use it? Why?

Reply:
I am fine to prune it, just did the union of both init...